src/test/java/be/seeseemelk/mockbukkit/entity/PlayerMockTest.java

```diff
@@ -1059,4 +1060,38 @@ public void testSimulateBlockPlaceInvalid()
 		player.setGameMode(originalGM);
 		assertFalse(worked);
 	}
+	@Test
```

Comment:
Minor styling remark
```suggestion
	}

	@Test
```